packages/shared/ReactFeatureFlags.js

```diff
@@ -15,6 +15,10 @@ export const enableFilterEmptyStringAttributesDOM = false;
 // Intended to enable React core members to more easily debug scheduling issues in DEV builds.
 export const enableDebugTracing = false;
 
+// Adds user timing marks for e.g. state updates, suspense, and work loop stuff,
+// for an experimental scheduling profiler tool.
+export const enableSchedulingProfiling = false;
```

Comment:
Sure, I'll rename this. Would we also want to rename the files from `SchedulingProfiling.js` to `SchedulingProfiler.js`? I'm just slightly concerned that it could mislead people into thinking that the file contains an actual profiler (like the DevTools Profiler).

Reply:
It would probably be nicer for the file and the feature flag to align, yes.

I don't think there's too much risk of confusion based on the name, since the package names are pretty clear :smile:

Reply:
Done!